scripts/auto-rebase/rebase-lvms.sh

```diff
@@ -153,15 +154,42 @@ download_lvms_operator_bundle_manifest(){
         local csv="lvms-operator.clusterserviceversion.yaml"
         local namespace="openshift-storage"
         extract_lvms_rbac_from_cluster_service_version "${PWD}" "${csv}" "${namespace}"
+        extract_lvms_deploy_from_cluster_service_version "${PWD}" "${csv}" "${namespace}"
 
         # Push the configMap to the kube-public namespace so that it is available to all users/apps
         generate_version_config_map "${version}" "lvms-version" "kube-public"\
             > "${PWD}/topolvm-configmap_lvms-version.yaml"
 
+        # Loop over all services and roles since they need to be patched with a namespace if they were not
+        # part of the CSV role definitions
+        for file in "${PWD}"/*; do
+          if [[ ${file} == *.yaml || ${file} == *.yml ]]; then
+              patch_namespace "Service" "${file}" "${namespace}"
+              patch_namespace "Role" "${file}" "${namespace}"
+              patch_namespace "RoleBinding" "${file}" "${namespace}"
+          fi
+        done
+
         popd || return 1
     done
 }
 
+patch_namespace() {
+  local kind=$1
+  local file=$2
+  local namespace=$3
+
+  if [[ $(yq e ".kind == \"${kind}\"" "${file}") == "true" ]]; then
+    # Check if the .metadata.namespace is not set or empty
+    if [[ $(yq e ".metadata.namespace == \"${namespace}\"" "${file}") == "false" ]]; then
+      echo "patching .metadata.namespace to \"${namespace}\" in ${file}"
+      # Set the .metadata.namespace to the specified value
+      yq e '.metadata.namespace = "'"${namespace}"'"' -i "${file}"
```

Comment:
Nit - The bashy quote wrapping could be cleaned up as:
`ns=$namespace yq e '.metadata.namespace = strenv(ns)' -n`